### DISCUSSION TITLE
Changing slide with Interactive IFrame Backgrounds in Revealjs slideshow

### DISCUSSION THREAD

**Discussion by cgadal** [quarto-dev/quarto-cli#4109] [Q&A] [answered]
Following the [documentation](https://quarto.org/docs/presentations/revealjs/#iframe-backgrounds), you can use a website as the slide background. You can also enable interactivity with this website using `background-interactive=True`:

```
---
title: "Minimal example"
format:
  revealjs
---

## Previous slide

##  {background-iframe="https://en.wikipedia.org/wiki/Red_panda" background-interactive=True}

## Next slide
```

If I don't interact with the website during the presentation, I can go from one slide to the next one. However, as soon as I start interacting with the interactive background-iframe, I can not use the keyboard anymore to go to the previous or next slide. 
How can I leave interactivity with the background-iframe during the presentation to change slide ?

**Comment by cscheid** [comment, marked as answer]
I don't think that's automatically possible, because you've selected the background frame and indicated that you wish to give the background frame focus.

You could try clicking on an element that's outside the iframe (like the title, for example). That will require, however, that you keep a visible element on the screen.

**Reply by cderv** [reply]
@Cgadal Among visible element, you'll have the menu icon which can allow you to move out to another slide. You could also activate control on screen (see about `controls: true` in https://quarto.org/docs/reference/formats/presentations/revealjs.html). We deactivate by default, put you can put it back, and you'll get some arrow to move between slides. They should stay even with the background.
